Sources/Apodini/WebService.swift

```diff
@@ -170,3 +176,8 @@ public enum WebServiceExecutionMode {
     /// It exits afterwards.
     case boot
 }
+
+private protocol CLIParsable {}
+extension Option: CLIParsable {}
+extension Argument: CLIParsable {}
+extension OptionGroup: CLIParsable {}
```

Comment:
What about Flag?

Reply:
Be sure to check the documentation of the ArgumentParser, maybe there are even more CLI types that I'm not aware of

Reply:
I agree with @philippzagar, we are currently missing a few types and there might even be more types added in the future. I would go with a different approach as detailed above and remove this protocol 👍